### PR TITLE
Per-place setting for 'no publication document' text

### LINKED ISSUE #1233: Change wording of unverified notice for law reform compiled consolidations
Change the wording of the notice on acts digitized from official consolidations: "[Note: The original publication document is not available and this content could not be verified.]" to wording that reflects the fact that this is an official and authorised (hence verified by the Law Reform Commission) version of legislation

### PR DESCRIPTION
**Pull request by goose-life**
closes #1233 

TODO: will `blocktrans` work here or always display the saved string? UPDATE: the latter, using `trans` instead

Form:
image

Coverpage English:
image

Coverpage Sotho:
image

### CODE REVIEW

File: indigo_api/templates/indigo_api/akn/coverpage_act.html [hunk 1]
```diff
@@ -192,7 +192,7 @@ <h2>{{ subt }}</h2>
         {% if not document.work.publication_document %}
           {% block verification_notice %}
             <li class="verification-notice">
-              [{% blocktrans %}Note: The original publication document is not available and this content could not be verified.{% endblocktrans %}]
+              [{% blocktrans with no_publication_document_text=document.work.place.settings.no_publication_document_text %}{{ no_publication_document_text }}{% endblocktrans %}]
```
Comment (longhotsummer) [inline indigo_api/templates/indigo_api/akn/coverpage_act.html:195]: I don't think blocktrans is quite right, see here: https://stackoverflow.com/questions/6546933/django-translate-variable-content-in-template. We want it to translate the contents of the variable, not treat it as a translatable string containing a variable.

We should also check if there is no message and not include this notice at all - ie. provide a way to turn it off.

File: indigo_api/models/places.py [hunk 2]
```diff
@@ -173,6 +173,7 @@ class PlaceSettings(models.Model):
     spreadsheet_url = models.URLField(null=True, blank=True)
     as_at_date = models.DateField(null=True, blank=True)
     styleguide_url = models.URLField(null=True, blank=True)
+    no_publication_document_text = models.CharField(max_length=1024, default='Note: The original publication document is not available and this content could not be verified.')
```
Comment (longhotsummer) [inline indigo_api/models/places.py:176]: `null=False, blank=True` ? So we always want a string, but it can be blank.

**Comment by goose-life** [comment]
Coverpage Sotho, after adding the following to st/LC_MESSAGES/django.po:
```
msgid "Note: The original publication document is not available and this content could not be verified."
msgstr "S Sotho translation"
```

image

**Review by longhotsummer — APPROVED** [review]

**Review by longhotsummer — CHANGES_REQUESTED** [review]
One more thing: in order for Django to realise that that string is translatable (and therefore present it for translation in CrowdIn), we need to mark it as translatable and [run makemessages](https://github.com/laws-africa/indigo#adding-translation-strings).

See details at https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#lazy-translation.

Basically, import `ugettext_lazy` as `_` and apply it to the default string in the model definition `_('Note: blah blah')`.

Then run makemessages as documented in Indigo's readme and check that it shows up.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**